From: Remy Bohmer <linux@bohmer.net>
Date: Sat, 29 Oct 2011 19:10:22 +0200
Subject: [PATCH] Fix compilation of dibbler package

A header file is included that is not needed, and even breaks
compilation on recent kernels. Remove it.

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 Port-linux/utils.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/Port-linux/utils.c b/Port-linux/utils.c
index 4bdda71..14a27f5 100644
--- a/Port-linux/utils.c
+++ b/Port-linux/utils.c
@@ -14,6 +14,7 @@
  * Rani Assaf <rani@magic.metawire.com> 980929:	resolve addresses
  */
 
+#define GNU_SOURCE
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
@@ -26,7 +27,7 @@
 #include <arpa/inet.h>
 #include <resolv.h>
 #include <asm/types.h>
-#include <linux/pkt_sched.h>
+/*#include <linux/pkt_sched.h>*/
 #include <time.h>
 #include <sys/time.h>
 
